### DISCUSSION TITLE
How to check if my bundles are compatibles with pimcore X? [EcommerceFramework]

### DISCUSSION THREAD

**Discussion by AlephLau** [pimcore/pimcore#13268] [Q&A / Support]
I will upgrade pimcore 6.9 to pimcore 10, but would like to know if my bundles are compatible with the upgrade.

I have the ecommerceframework bundle version 6.9 on my pimcore 6.9, should I deactivate or uninstall the this bundle to upgrade to pimcore 10 or does it update automatically when I upgrade via composer?

Can I deactivate the bundles and then upgrade to pimcore 10, to finally activate them and check if they work or what is the recommended procedure?

I would really appreciate your opinions and how you think it is the best way to work with bundles when upgrading from pimcore 6.9 to pimcore 10

**Comment by kingjia90** [comment]
The EcommerceFramework is a built-in bundle of Pimcore and it gets updated along with it. AFAIK there are no any extra steps required.


NB: https://github.com/pimcore/pimcore/issues/12526, from 11, it may change and be more independent and upgradeable from composer.
